_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,155 @@
+{: .tip}
+
+## Installing on Linux and macOS
+
+If you want to run OpenSearch Benchmark in a Docker container, see [Installing with Docker](#installing-with-docker). The OpenSearch Benchmark Docker image includes all of the required software, so there are no additional steps for you to take.
+{: .important}
+
+To install OpenSearch Benchmark directly on a UNIX host such as Linux or macOS, make sure you have **Python 3.8 or newer** installed. 
+
+If you need help installing Python, refer to the official [Python Setup and Usage](https://docs.python.org/3/using/index.html) documentation. 
+
+### Checking software dependencies
+
+Before you being installing OpenSearch Benchmark, check the following software dependencies.
+
+Use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of Python on your host. This is especially useful if your "system" version of Python is older than version 3.8.
+{: .tip}
+
+- Check that Python 3.8 or newer is installed:
+
+  ```bash
+  python3 --version
+  ```
+  {% include copy.html %}
+
+- Check that `pip` is installed and functions:
+
+  ```bash
+  pip --version
+  ```
+  {% include copy.html %}
+
+- _Optional_ Check that your installed version of `git` is **Git 1.9 or newer**. `git` is not required for OpenSearch Benchmark installation, but it is required to fetch benchmark workload resources from a repository when you want to perform tests. See the official Git [Documentation](https://git-scm.com/doc) for help installing Git. 
+
+  ```bash
+  git --version
+  ```
+  {% include copy.html %}
+
+### Completing the installation
+
+After the prerequisites are installed, you can install OpenSearch Benchmark using the following command:
+
+```bash
+pip install opensearch-benchmark
+```
+{% include copy.html %}
+
+After the installation completes you can use the following command to display help information:
+
+```bash
+opensearch-benchmark -h
+```
+{% include copy.html %}
+
+
+Now that OpenSearch Benchmark is installed on your host you can learn about [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/).
+
+## Installing with Docker
+
+You can find the official Docker images for OpenSearch Benchmark on [Docker Hub](https://hub.docker.com/r/opensearchproject/opensearch-benchmark) or on the [Amazon ECR Public Gallery](https://gallery.ecr.aws/opensearchproject/opensearch-benchmark).
+
+
+### Docker limitations
+
+Some OpenSearch Benchmark functionality is unavailable when you run OpenSearch Benchmark in a Docker container. Specifically, the following restrictions apply:
+
+- OpenSearch Benchmark cannot distribute load from multiple hosts, such as load worker coordinator hosts.
+- OpenSearch Benchmark cannot provision OpenSearch nodes and can only run tests on previously existing clusters. You can only invoke OpenSearch Benchmark commands using the `benchmark-only` pipeline.
+
+### Pulling the the Docker images
+
+To pull the image from Docker Hub:
```

Comment:
Let's complete the sentence: "To pull the image from Docker Hub, run the following..."